cli/fabric.c

```diff
@@ -37,6 +38,8 @@
 #include <errno.h>
 #include <ctype.h>
 
+#if ENABLE_FABRIC_MENU
```

Comment:
Nak.

This is a horrible idea. Compile time options should only be added when there is an optional *dependency*. 

Don't add them for silly policy reasons. Downstream users (think distro's who want to distrobute *binaries*) will hate you, or just enable them all by default. Don't make people re-compile for these reasons. 

A built binary should have all possible features that you can provide given the compile environment.

Reply:
The 'fabric' menu needs to be removed from the upcoming PSX release because fabric commands are not supported by PSX devices. 

The PAX release (which supports fabric commands) is still a few months out, and in that release, we will remove the #ifdef so user can have the full feature (regardless of whether they're using PSX or PAX). 

At this moment, we don't want to distract PSX user with 'fabric' submenu commands and create more explaining work for the support team. 

@lsgunth let me know what you think.

Reply:
If you want to do a release without parts of the code then create a PSX branch without those patches and release from there.

Reply:
Creating a new branch and release without the ~10 fabric-related commits is doable, but it incurs more effort in development and testing than previously planned, since the commits could affect other files in the projects, not just the fabric.c/h. 

While the "#if ENABLE_FABRIC_MENU" does look slightly out of place, considering the current RTP release deadline, I think it provides a good trade off between code quality and test effort, and we maintain our tradition of releasing from 'master' branch as well.

Reply:
It makes more sense to do the release branch, isn't actually any more effort (really it's the same testing seeing you're either removing the code through an #if or via a commit). If you want to do the branch with a new commit that just removes all the code between the #ifs, that's fine too.

Adding the compile time define just leaves it all in for people to use anyway and isn't what you want because it will have to be removed later anyway (ie. more work).

Reply:
We will create a new branch for PSX/PFX release. This commit has been dropped in this PR.